packages/lucide-react/package.json

```diff
@@ -65,6 +65,6 @@
     "vitest": "^1.1.1"
   },
   "peerDependencies": {
-    "react": "^16.5.1 || ^17.0.0 || ^18.0.0 || ^19.0.0"
+    "react": "^16.5.1 || ^17.0.0 || ^18.0.0 || ^19.0.0-rc"
```

Comment:
Should it not support both the pre-release and normal release?

Otherwise people with be forced to update the package again even if they don't really need to.

Reply:
@VIKTORVAV99 We made a mistake the specific version `19.0.0` does not exists yet. Only `react@19.0.0-rc`. So the change of @pantoninho is right

Reply:
Sorry for the late reply.
I did not include `19.0.0` because I can't confidently say that `lucide-react` will be compatible with `react@19.0.0`. I thought it would be worse if it was indeed incompatible and broke installations that upgraded to `react@19.0.0`.

That was my line of thought, but I agree with you that the API is most likely stable. If you want I can open a PR that includes both versions.